lnst/Common/IpAddress.py

```diff
@@ -126,6 +130,16 @@ def is_tentative(self):
         IFA_F_TENTATIVE = 0x40
         return IFA_F_TENTATIVE & self.flags
 
+
+def interface_addresses(network: Union[IPv4Network, IPv6Network]) -> Generator[BaseIpAddress]:
```

Comment:
The return type should either be the full `Generator[BaseIpAddress, None, None]` or you should use `Iterator[BaseIpAddress]` instead, because I doubt we're going to use the generator's `send()` method.

Reply:
Please ammend the fix in the commit that adds this function. We want to have a bisectable commit history and having fixup commits added on top of a patch set would break this.